Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -1988,12 +1978,6 @@ script:
     - contextPath: Gmail.ForwardingAddress.verificationStatus
       description: Indicates whether this address has been verified and is usable for forwarding.
       type: String
-    - contextPath: Gmail.ForwardingAddress.Disposition
-      description: The state that a message should be left in after it has been forwarded.
-      type: String
-    - contextPath: Gmail.ForwardingAddress.Enabled
```

Comment:
The enabled is part of the disposition? not the add?

Reply:
Yes, it part of update, attached API doc: 
https://developers.google.com/gmail/api/reference/rest/v1/AutoForwarding